getting-started/faqs.md

```diff
@@ -0,0 +1,101 @@
+# Frequently Asked Questions
+
+## Business Users
+
+Design questions to influence purchasing decision
```

Comment:
we shouldn't be telling users this. Can we remove the Business Users section and go directly to About CFT?